ulacitbnb/Controllers/AccomodationController.cs

```diff
@@ -14,10 +15,9 @@ namespace ulacit_bnb.Controllers
     [RoutePrefix("api/accomodation")]
     public class AccomodationController : ApiController
     {
-
         //SQL Connection
-        SqlConnection sqlConnection = new SqlConnection(ConfigurationManager.ConnectionStrings["UlacitbnbAzureDB"].ConnectionString);
-
+        SqlConnection sqlConnection = ConnectionString.GetSqlConnection();
```

Comment:
instead of storing the connection string on a variable on each class, call the `GetSqlConnection()` on the methods... this way the connection pooling works as expected